script/pktgen_dtap0.sh

```diff
@@ -14,10 +14,12 @@ function add_dtap() {
 }
 
 function set_dtap() {
-    echo "min_pkt_size 64" > /proc/net/pktgen/dtap0
-    echo "max_pkt_size 9000" > /proc/net/pktgen/dtap0
-    echo "count 20" > /proc/net/pktgen/dtap0
+    echo "min_pkt_size 90000" > /proc/net/pktgen/dtap0
+    echo "max_pkt_size 90000" > /proc/net/pktgen/dtap0
+    # count 0 means transmitting the packets continuously.
+    echo "count 0" > /proc/net/pktgen/dtap0
     echo "dst ${TARGET_IP}" > /proc/net/pktgen/dtap0
+    echo "rate 10000M" > /proc/net/pktgen/dtap0
 }
 
```

Comment:
In addition, the most important point to us is the core affinity on packet generator, to avoid the resource conflict with `gpuflow` program ( at least 0-2 cores). How can I achieve this with packet generator? If can't, a simple way is to add a cgroup (cpuset) limitation.